docs/presets.md

```diff
@@ -0,0 +1,104 @@
+# Gasket Preset Authoring Guide
+
+## Background
+
+Much like [babel presets], Gasket presets allow common plugins to be grouped and
+loaded together. They serve 2 purposes: to serve as codified sets of plugins,
+and to facilitate rapid creation of Gasket application.
+
+At GoDaddy, we have presets specifically tailored with internal sets of plugins,
+making maintaining standards around authentication, style, analytics, and more
+significantly easier.
+
+See the [naming conventions] for how to best name a preset, ensuring that
+Gasket's plugin engine properly resolves it.
```

Comment:
This whole guide needs to be quite a bit different now. Presets are no longer like babel's, as they are only used to provide configuration and prompting to facilitate app generation using `create-gasket-app`.

If you want to take a stap at reworking this doc now, go for it. Otherwise, spin up a separate ticket for us to address as a follow up.